# April Tag Detection Array Message
# Array of April Tag detections

std_msgs/Header header          # Standard message header
AprilTagDetection[] detections  # Array of detections
int32 num_detections            # Number of detections
float64 processing_time_ms      # Processing time in milliseconds